Makefile

```diff
@@ -31,4 +33,8 @@ start:
 
 .PHONY: stop
 stop:
-	docker-compose -f server/conf/docker/docker-compose-dev.all.debug.yml down
\ No newline at end of file
+	docker-compose -f server/conf/docker/docker-compose-dev.all.debug.yml down
+
+.PHONY: stop-full
+stop-full:
```

Comment:
maybe something more explicit like `stop-and-remove-volumes`

Reply:
more explicit is nice, but I think we also want it to be succinct so it still saves time

Reply:
maybe `stop-v`?

Reply:
I like stop-v.